### PR TITLE
Weather data tutorial

### PR DESCRIPTION
**Pull request by domoritz**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -13,6 +13,7 @@
     "test": "test"
   },
   "scripts": {
+    "postinstall": "cp node_modules/vega-datasets/data/* data",
```
Comment (kanitw) [inline package.json]: (just complaining) This PR is quite convoluted.

Comment (domoritz) [inline package.json]: Ack. I needed a new dataset and added this change.

File: site/tutorials/weather.md [hunk 2]
```diff
@@ -1,36 +1,204 @@
...
+    }
+  }
+}
+</div>
+
+This chart shows that in Seattle the precipitation in the winter months is much higher than during the summer. You can now create similar charts for the other variables on your own!
+
+When looking at precipitation and temperature, we might want to aggregate by year and month rather than just month. This allows us to see seasonal trends but for each year separately. You can find more about [time units in the documentation]({{site.baseurl}}/docs/timeUnit.html). Note that we set the aggregation to `max`so that we see the maximum temperature in each month.
+
+<div class="vl-example">
```
Comment (kanitw) [inline site/tutorials/weather.md]: The label is quite ugly here.  Rotate label?

File: site/tutorials/weather.md [hunk 3]
```diff
@@ -1,36 +1,204 @@
...
+      "aggregate": "max"
+    }
+  },
+  "config": {
+    "unit": { "width": 300 }
+  }
+}
+</div>
+
+In this chart it looks as though the maximum temperature is increasing from year to year. To confirm this, let's create a chart that shows the mean of the maximum daily temperatures.
```
Comment (kanitw) [inline site/tutorials/weather.md]: "To confirm" sounds like we should run a statistical test to me.

File: site/tutorials/weather.md [hunk 4]
```diff
@@ -1,36 +1,204 @@
...
+
+<div class="vl-example">
+{
+  "data": {"url": "data/seattle-weather.csv", "formatType": "csv"},
+  "mark": "point",
```
Comment (kanitw) [inline site/tutorials/weather.md]: why point this time?

File: site/tutorials/weather.md [hunk 5]
```diff
@@ -1,36 +1,204 @@
...
-* **WT18** - Snow, snow pellets, snow grains, or ice crystals
-* **WT08** - Smoke or haze
-* **WT11** - High or damaging winds
-* **WT22** - Ice fog or freezing fog
-* **WT01** - Fog, ice fog, or freezing fog (may include heavy fog)
-* **WT02** - Heavy fog or heaving freezing fog (not always distinguished from fog)
-* **WT13** - Mist
-
--->
+This is the end of this tutorial. You can find more visualizations in the [gallery]({{site.baseurl}}/gallery.html).
```
Comment (kanitw) [inline site/tutorials/weather.md]: This is a very abrupt way to end a tutorial.  ><"

File: site/tutorials/weather.md (hunk 4 above)
Comment (domoritz) [inline site/tutorials/weather.md]: Line has the problem that we actually don't have much data in between. And we don't have point on line.

File: site/tutorials/weather.md (hunk 3 above)
Comment (kanitw) [inline site/tutorials/weather.md]: @domoritz  this comment is ignored.

Comment (domoritz) [inline site/tutorials/weather.md]: Hmmpf, I fixed this but apparently overwrote when I merged.

Comment (domoritz) [inline site/tutorials/weather.md]: Rename to "To look closer into this", okay?

**Pull request merged**